pyproject.toml

```diff
@@ -36,6 +36,7 @@ python = "^3.7"
 celery = "~5.2.3"
 # Fundamental web framework for Nautobot
 # NOTE: some of our dependencies do not have releases compatible with Django 3.2's DEFAULT_AUTO_FIELD setting
+# but these dependencies now have workarounds in our current code base
```

Comment:
Is this even necessary anymore? If our dependencies have workarounds for `3.2` doesn't that mean they are compatible?

Reply:
Good point!